tests/testthat/test-oauth-flow-auth-code.R

```diff
@@ -51,16 +51,20 @@ test_that("bare authorisation codes can be input manually", {
 # normalize_redirect_uri --------------------------------------------------
 
 test_that("adds port to localhost url", {
+  skip_if(is_hosted_session(), "Skipping in hosted sessions")
```

Comment:
I think it would be slightly better just to mock these out, i.e.

```R
local_mocked_bindings(is_hosted_session = function() FALSE)
```

And then if you were really keen you could add a specific test for the error when it's `TRUE`. (But if you don't have time I can do it too)

Reply:
Done.